src/features/country/Country.js

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect } from 'react';
+import { FiArrowRightCircle } from 'react-icons/fi';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link, useParams } from 'react-router-dom';
+import europe from '../continent/europe.png';
+import './country.css';
+import { fetchCountry } from './countryReducer';
+
```

Comment:
- [ ] Kindly implement a **redux filter/ search** functionality on your Homepage so as to enable filtering of countries.